google/resource_dataproc_cluster.go

```diff
@@ -310,6 +310,21 @@ func resourceDataprocCluster() *schema.Resource {
 								},
 							},
 						},
+						"encryption_config": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"key_name": {
```

Comment:
Since this field will only support keys that have been stored in google KMS, can you rename to `kms_key_name` please?